extensions/mssql/package.json

```diff
@@ -367,6 +367,16 @@
           "type": "boolean",
           "description": "%mssql.ignorePlatformWarning%",
           "default": false
+        },
+        "mssql.useKeytarCredential": {
+          "type": "boolean",
+          "description": "%mssql.useKeytarCredential%",
+          "default": true
+        },
+        "mssql.migrateLinuxCredential": {
```

Comment:
Hm, why are you having this as a setting? That means this will show up in the settings pane even if they're not on a linux machine or don't have any credentials to migrate.